src/components/Spawn/Layout.tsx

```diff
@@ -101,3 +101,5 @@ export const PaddedButton = styled(Button)`
   margin-right: ${size.xxs};
   flex-grow: 0;
 `;
+
+export const tooltipWidth = "250px";
```

Comment:
I gave a width to all of the tooltips because [PD-1542](https://jira.mongodb.org/browse/PD-1542) is still a problem, leading to weird behaviors like:

<img width="1553" alt="Screen Shot 2022-03-15 at 3 57 27 PM" src="https://user-images.githubusercontent.com/47064971/159030644-67fe2fae-f444-4892-aa98-08d5e14dbe87.png">

<img width="1553" alt="Screen Shot 2022-03-15 at 3 57 36 PM" src="https://user-images.githubusercontent.com/47064971/159030668-3820918a-59d4-44bb-a734-248fe9b2333c.png">